e2e/nextjs/tsconfig.json

```diff
@@ -19,5 +19,9 @@
     {
       "path": "./tsconfig.lib.json"
     }
-  ]
+  ],
+  "ts-node": {
+    "require": ["tsconfig-paths/register"],
+    "swc": true
+  }
```

Comment:
I switched to `ts-node` instead of `swc-node` as Nx uses it. 
`ts-node` suggests to keep config in `tsconfig.json` files 
 https://typestrong.org/ts-node/docs/swc/